lib/domain/repository/question.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:job_interview_practice/data/models/question.dart';
+import 'package:job_interview_practice/domain/entities/question.dart';
+
+abstract class QuestionRepository extends Question{
```

Comment:
Why does this extend `Question` (or anything)? 

See the PR comment for line 3 of `lib/data/models/question.dart`.